charts/longhorn/v1.1.3/values.yaml

```diff
@@ -9,22 +9,22 @@ image:
   longhorn:
     engine:
       repository: longhornio/longhorn-engine
```

Comment:
these longhorn images should all be mirrored

Reply:
Just use upstream images as before.